docs/source/en/api/pipelines/pixart_sigma.md

```diff
@@ -0,0 +1,153 @@
+
+You can find the original codebase at [PixArt-alpha/PixArt-sigma](https://github.com/PixArt-alpha/PixArt-sigma) and all the available checkpoints at [PixArt-alpha](https://huggingface.co/PixArt-alpha).
+
+Some notes about this pipeline:
+
+* It uses a Transformer backbone (instead of a UNet) for denoising. As such it has a similar architecture as [DiT](./dit).
+* It was trained using text conditions computed from T5. This aspect makes the pipeline better at following complex text prompts with intricate details.
+* It is good at producing high-resolution images at different aspect ratios. To get the best results, the authors recommend some size brackets which can be found [here](https://github.com/PixArt-alpha/PixArt-sigma/blob/master/diffusion/data/datasets/utils.py).
+* It rivals the quality of state-of-the-art text-to-image generation systems (as of this writing) such as PixArt-α, Stable Diffusion XL, Playground V2.0 and DALL-E 3, while being more efficient than them.
+* It shows the ability of generating super high resolution images, such as 2048px or even 4K.
+* It rivals that the Text-to-Image models can grow from a weak model to a stronger one through several improvements(VAEs, Datasets and so on.)
+
+<Tip>
+
+Make sure to check out the Schedulers [guide](../../using-diffusers/schedulers.md) to learn how to explore the tradeoff between scheduler speed and quality, and see the [reuse components across pipelines](../../using-diffusers/loading.md#reuse-a-pipeline) section to learn how to efficiently load the same components into multiple pipelines.
+
+</Tip>
+
+## Inference with under 8GB GPU VRAM
+
+Run the [`PixArtSigmaPipeline`] with under 8GB GPU VRAM by loading the text encoder in 8-bit precision. Let's walk through a full-fledged example. 
+
+First, install the [bitsandbytes](https://github.com/TimDettmers/bitsandbytes) library:
+
+```bash
+pip install -U bitsandbytes
+```
+
+Then load the text encoder in 8-bit:
+
+```python
+from transformers import T5EncoderModel
+from diffusers import PixArtSigmaPipeline
+import torch
+
+text_encoder = T5EncoderModel.from_pretrained(
+    "PixArt-alpha/PixArt-Sigma-XL-2-1024-MS",
+    subfolder="text_encoder",
+    load_in_8bit=True,
+    device_map="auto",
+
+)
+pipe = PixArtSigmaPipeline.from_pretrained(
+    "PixArt-alpha/PixArt-Sigma-XL-2-1024-MS",
+    text_encoder=text_encoder,
+    transformer=None,
+    device_map="auto"
+)
+```
+
+Now, use the `pipe` to encode a prompt:
+
+```python
+with torch.no_grad():
+    prompt = "cute cat"
+    prompt_embeds, prompt_attention_mask, negative_embeds, negative_prompt_attention_mask = pipe.encode_prompt(prompt)
```

Comment:
```suggestion
    prompt_embeds, prompt_attention_mask, negative_embeds, negative_prompt_attention_mask = pipe.encode_prompt(prompt, max_sequence_length=300)
```
I got a warning that I was exceeding the 150 token limit and the prompt was truncated

Reply:
Are you using the PixArtSigmaPipeline, cause the default value of max_sequence_length is 300.
https://github.com/lawrence-cj/diffusers/blob/d5f4c217b5f6ac4fd23b4ee627611e0fbe0707a7/src/diffusers/pipelines/pixart_alpha/pipeline_pixart_sigma.py#L619

Reply:
yeah I'm using it, but here we're not using the encode prompt with the pipeline, we're using it directly and the default is 120 (I don't really remember if the error said 150 or 120 though): https://github.com/lawrence-cj/diffusers/blob/d5f4c217b5f6ac4fd23b4ee627611e0fbe0707a7/src/diffusers/pipelines/pixart_alpha/pipeline_pixart_sigma.py#L215

When using the pipeline in a normal way, the ` max_sequence_length` is also passed when encoding the prompt: https://github.com/lawrence-cj/diffusers/blob/d5f4c217b5f6ac4fd23b4ee627611e0fbe0707a7/src/diffusers/pipelines/pixart_alpha/pipeline_pixart_sigma.py#L759

So one solution would be to change this behavior in the code instead of the docs.

Reply:
Oh, you are right. Maybe it's better to change the default value of it to 300.

Reply:
agree - I think we should keep the default value same for pipeline and `encode_prompt`

Reply:
fixed here: https://github.com/huggingface/diffusers/pull/7857/commits/502313b4a5827cd1408b526b65e418beb0fedc91